src/lib/input/autosize.ts

```diff
@@ -71,9 +71,23 @@ export class MdTextareaAutosize implements AfterViewInit {
 
   ngAfterViewInit() {
     this._cacheTextareaLineHeight();
+    this._setInitialRows();
     this.resizeToFitContent();
   }
 
+  /**
+   * Sets the rows property of the textarea to one by default.
+   * Browsers by default set the rows property to two. Therefore setting the minRows and maxRows
+   * binding to something below two doesn't work.
+   */
+  private _setInitialRows() {
```

Comment:
Done. Not sure why the screenshot tests fail. The images don't seem to be different.